be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/validators/SocialMediaValidator.scala

```diff
@@ -0,0 +1,66 @@
+package ch.epfl.pop.pubsub.graph.validators
+
+import ch.epfl.pop.model.network.JsonRpcRequest
+import ch.epfl.pop.model.network.method.message.Message
+import ch.epfl.pop.model.network.method.message.data.socialMedia.{AddChirp}
+import ch.epfl.pop.model.objects.{Hash, Base64Data, PublicKey, Channel}
+import ch.epfl.pop.pubsub.graph.{GraphMessage, PipelineError, DbActor}
+import ch.epfl.pop.model.network.method.message.data.{ActionType, ObjectType}
+import ch.epfl.pop.model.network.method.message.data.rollCall.CloseRollCall
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.{Await, Future}
+
+
+
+case object SocialMediaValidator extends MessageDataContentValidator with EventValidator {
+    override def EVENT_HASH_PREFIX: String = "SocialMedia" //to check later on
```

Comment:
Is this not defined yet? If not, I'd suggest creating an issue to discuss it with other members. Moreover, we should (I hope) have a list of these prefixes somewhere in the protocol

Reply:
No, as it was created this semester. Indeed, it should be discussed

Reply:
No, as it was created this semester. Indeed, it should be discussed

Reply:
Actually, there is no need for this, at least for the current social media messages

Reply:
We just need to override it with something (for now) because of what it extends

Reply:
Or maybe I should remove the eventvalidator trait for now